
Well, here in Germany (Berlin) I need the 7Mhz frequency ...

  - Rene Rebe <rene@exactcode.de>

--- dvbd-0.7.3/src/dvbttuneparams.cpp.orig	2004-08-21 10:18:27.353990887 +0200
+++ dvbd-0.7.3/src/dvbttuneparams.cpp	2004-08-21 10:18:59.155921855 +0200
@@ -69,6 +69,7 @@
     transTable["GUARD_INTERVAL_1_16"] = GUARD_INTERVAL_1_16;
     transTable["GUARD_INTERVAL_1_32"] = GUARD_INTERVAL_1_32;
     transTable["HIERARCHY_NONE"] = HIERARCHY_NONE;
+    transTable["BANDWIDTH_7_MHZ"] = BANDWIDTH_7_MHZ;
     transTable["BANDWIDTH_8_MHZ"] = BANDWIDTH_8_MHZ;
     transTable["TRANSMISSION_MODE_2K"] = TRANSMISSION_MODE_2K;
     transTable["TRANSMISSION_MODE_8K"] = TRANSMISSION_MODE_8K;
